controller/state.go

```diff
@@ -141,7 +151,8 @@ func (m *appStateManager) getRepoObjs(app *v1alpha1.Application, source v1alpha1
 		KustomizeOptions: &appv1.KustomizeOptions{
 			BuildOptions: buildOptions,
 		},
-		KubeVersion: cluster.ServerVersion,
+		KubeVersion:          cluster.ServerVersion,
+		AdditionalHelmValues: additionalHelmValues,
```

Comment:
This is really just a string that's being passed to the repo-server as an additional file to use when running helm template.  If we wanted to make this a little more generic, we should rename this property and it could then just be used appropriately based on whatever type the application is (helm, kustomize, jsonnet, etc..)